### ISSUE TITLE
Prevent double submissions

### ISSUE THREAD

**Issue by zregvart** [syndesisio/syndesis-ui#1113]
Its an issue when creating connections using OAuth dance, _I think_ first request defines the connection based on the users input (name) and the client state (in cookie), that proceeds to remove the client state cookie, and then the second request overwrites the first but without the OAuth data from the client state as the cookie is missing at this point.
But nonetheless we should prevent the user in initiating a second request using the same trigger (button) while the processing of the first request.

**Comment by gashcrumb** [comment]
So disable the `verify` button right when the user clicks on it?

**Comment by zregvart** [comment]
> So disable the verify button right when the user clicks on it?

Yeah, too bad patternfly doesn't have a spinner on buttons OOTB... 👀

**Comment by gashcrumb** [comment]
That's no problem.  I'm thinking one option is to just set a toggle in the controller and stick the button in a  `<syndesis-loading>` which would then replace the button with a spinner when it's clicked on.

**Comment by zregvart** [comment]
Also root cause could be this https://github.com/syndesisio/syndesis-rest/issues/741
